css-layout-classes/index.html

```diff
@@ -0,0 +1,88 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>CSS Grid Classes</title>
+    <link rel="stylesheet" href="styles.css">
+  </head>
+  <body>
+    <div class="container">
+      <div class="row">
+        <div class="column-full">
+          <h1>Choose your character!</h1>
+        </div>
+      </div>
+      <div class="row">
+        <div class="column-one-third">
+          <img src="images/link.png">
+        </div>
+        <div class="column-one-third">
+          <h2>Super Smash Bros. Ultimate</h2>
+          <p>Super Smash Bros. Ultimate received universal acclaim from both
+            critics and players, with some critics calling it the best
+            installment in the series.  It received praise for its large amount
+            of content and fine-tuning of existing Smash gameplay elements,
+            although its online mode was widely criticized.
+          </p>
+        </div>
+        <div class="column-one-third">
+          <img src="images/yoshi.png">
+        </div>
+      </div>
+      <div class="row">
+        <div class="column-full">
+          <div class="image-container">
+            <img src="images/kirby.png">
+          </div>
+          <div class="image-container">
+            <img src="images/fox.png">
+          </div>
+          <div class="image-container">
+            <img src="images/pikachu.png">
+          </div>
+          <div class="image-container">
+            <img src="images/ness.png">
+          </div>
+        </div>
```

Comment:
Instead of one column with four images in it, this should be four columns, each with one image in it.